Simplified/Book/Models/NYPLBook+Additions.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  NYPLBook+Additions.swift
+//  SimplyE
+//
+//  Created by Ettore Pasquini on 7/9/20.
+//  Copyright © 2020 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+extension NYPLBook {
```

Comment:
NB this is not actually new of course. Previously it was in NYPLBook+Logging.swift.